content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,110 @@
+
+<!-- It is already supported on the 5 latest Kubernetes versions and the community commit to maintain support for the most recent 5 Kubernetes minor versions. -->
+
+### Expressiveness
+The Gateway API provides standard, Kubernetes-backed support for a wide range of features, such as header-based matching, traffic splitting, weight-based routing, request mirroring and more. With Ingress, these features need custom provider-specific annotations.
+
+### Extensibility
+Gateway API is designed with extensibility as a core feature. Rather than enforcing a one-size-fits-all model, it offers the flexibility to link custom resources at multiple layers within the API's framework. This layered approach to customization ensures that users can tailor configurations to their specific needs without overwhelming the main structure. By doing so, the Gateway API facilitates more granular and context-sensitive adjustments, allowing for a fine-tuned balance between standardization and adaptability. This becomes particularly valuable in complex cloud-native environments where specific use cases require nuanced configurations.
+
+
+## Upgrading to Gateway
+
+1. [Install a Gateway controller](https://gateway-api.sigs.k8s.io/guides/#installing-a-gateway-controller) OR [install the Gateway API CRDs manually](https://gateway-api.sigs.k8s.io/guides/#installing-gateway-api) 
+
+2. install [ingress2gateway](https://github.com/kubernetes-sigs/ingress2gateway)
+
+```
+go install github.com/kubernetes-sigs/ingress2gateway@latest
+```
+Provided you have a Go development environment installed, that command installs `ingress2gateway` to `$(go env GOPATH)/bin/ingress2gateway`.
+
+Alternatively, you can build it from source by following [Build from source](https://github.com/kubernetes-sigs/ingress2gateway/tree/95b4a0316bac8d28d72b871a210bbc9bd6191eb7#build-from-source)
+
+3. Once the tool is installed, you can use it to convert the ingress resources in your cluster to GatewayAPI resources.
+
+```
+ingress2gateway print
+```
+
+This above command will:
+
+1. Read your Kube config file to extract the cluster credentials and the current active namespace.
+1. Search for ingresses and provider-specific resources in that namespace.
+1. Convert them to Gateway-API resources (Currently only Gateways and HTTPRoutes).
+For other options refer run the tool with `-h` or refer to https://github.com/kubernetes-sigs/ingress2gateway#options
+
+4. Review the converted Gateway API resources, validate them, and then apply them to your cluster.
+
+5. Update your DNS to point to the new gateway. 
+
+Note: make sure you test the newly created gateway before shifting the shifting 100% of the traffic.
```

Comment:
I'd recommend adding this as a step between 4 and 5. Potentially with an example like `curl -h Host:example.com HTTP://{GATEWAY_ADDRESS}/foo/bar` that shows people how to get an address from Gateway status.